playbook/app/pb_kits/playbook/pb_body/docs/_body_truncate.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import Body from '../_body';
+import Caption from '../../pb_caption/_caption'
+import Flex from '../../pb_flex/_flex'
+
+const BodyTruncate = (props) => {
+  const lorem = "Lorem ipsum dolor sit amet consectetur adipisicing elit. Veritatis, minus. Nisi beatae voluptatum labore sequi. Nemo accusantium corrupti, reiciendis magnam tenetur perferendis esse pariatur voluptas eaque hic vel rem nihil quidem dolorum ex dolor, libero ullam placeat, sapiente eos. Cumque obcaecati dignissimos molestiae, minima quibusdam sint maxime libero accusantium animi quis quia maiores enim ipsum, esse, modi laudantium illum error!"
+
+  return (
+    <Flex
+        maxWidth="md"
+        orientation="column"
+    >
+      <Caption text="After first row" />
```

Comment:
Code looks great @gavhuang ! 

Minor feedback: the Caption kit in this doc example + the title doc example is not showing dark mode styles when dark mode is activated. You'll have to pass in {...props} to Caption kit as well, same way you're passing it to the Body kit. The rails side dark mode works in a different way so that is already showing the correct dark mode styles so adding this in will make sure both react and rails doc examples are consistent!